### PR TITLE
Add option to disable incomplete classpath warning. Fixes #107

### LINKED ISSUE #107: Warning: Classpath is incomplete. Only syntax errors will be reported.
I set `java.home` to `"/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home"`, when I open a java file, the vscode shows the warning message **"Classpath is incomplete. Only syntax errors will be reported."**, what's wrong? How can I fix this issue?

This warning message is from this file: https://github.com/gorkem/java-language-server/blob/74efc7ac1d9c9abb6728864111e33b1eb4d5e483/org.jboss.tools.vscode.java/src/org/jboss/tools/vscode/java/internal/handlers/DocumentLifeCycleHandler.java#L114

##### Environment
- Operating System: macOS Sierra 10.12.1
- JDK version: 1.8.0_102
- Visual Studio Code version: 1.7.2
- Java extension version: 1.8.0_102

##### Steps To Reproduce
1. on `settings.json` file, set `java.home` to `"/Library/Java/JavaVirtualMachines/jdk1.8.0_102.jdk/Contents/Home"`
2. reload window
3. open a java file

### PR DESCRIPTION
**Pull request by fbricon**
Depends on https://github.com/gorkem/java-language-server/pull/112

Signed-off-by: Fred Bricon <fbricon@gmail.com>

### CODE REVIEW

**Comment by fbricon** [comment]
New endpoint added, @gorkem  can you please review?

File: src/extension.ts [hunk 1]
```diff
@@ -95,42 +95,76 @@ export function activate(context: ExtensionContext) {
...
+				break;
 		}
 		item.command = 'java.open.output';
 		item.tooltip = report.message;
 		toggleItem(window.activeTextEditor, item);
 	});
+	languageClient.onNotification(ActionableNotification.type, (notification) => {
+		let show = null;
+		switch (notification.severity) {
+			case MessageType.Log:
```
Comment (gorkem) [inline src/extension.ts]: Should it show a message for Log requests ?

File: src/extension.ts [hunk 2]
```diff
@@ -95,42 +95,76 @@ export function activate(context: ExtensionContext) {
...
+				show = window.showInformationMessage;
+				break;
+			case MessageType.Warning:
+				show = window.showWarningMessage;
+				break;
+			case MessageType.Error:
+				show = window.showErrorMessage;
+				break;
+		}
+		if (show === null) {
```
Comment (gorkem) [inline src/extension.ts]: `if( !show )`

File: src/extension.ts [hunk 3]
```diff
@@ -95,42 +95,76 @@ export function activate(context: ExtensionContext) {
...
-		if(report.type === 'Started'){
-			item.text = '$(thumbsup)';
-			lastStatus = item.text;
-		} else if(report.type === 'Error'){
-			item.text = '$(thumbsdown)';
-			lastStatus = item.text;
-		} else if(report.type === 'Message') {
-			item.text = report.message;
-			setTimeout(()=> {item.text = lastStatus;}, 3000);
+		switch (report.type) {
```
Comment (gorkem) [inline src/extension.ts]: Nice!!

File: src/extension.ts (hunk 1 above)
Comment (fbricon) [inline src/extension.ts]: Probably not, but if there are actions associated with it, we prolly want to show them

**Pull request merged**